### PR TITLE
added interaction icons

### PR DESCRIPTION
**Pull request by Flryang**
modified the prefab UI_canvas and added icons to the bottom right corner, which will display a hand icon whenever the target object is of type MOVE, PICKUP, OPEN, CLOSE and will display an eye icon whenever the target object is of type INTERACT, and will display a reflection icon whenever the player is able to teleport, otherwise it should be hidden.

### CODE REVIEW

**Review by ZuhabWasim — APPROVED** [review]

File: Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -297,7 +300,22 @@ void DisplayInteractionPrompts()
             bp2.hidePrompt();
         }
     }
-
+    void showInteractionIcon()
```
Comment (Lakshya2610) [inline Assets/Scripts/PlayerController.cs]: Can you follow the style convention for naming functions. It should be `ShowInteractionIcon`

File: Assets/Scripts/PlayerController.cs [hunk 2]
```diff
@@ -56,6 +57,7 @@ void Start()
         targetObject = null;
         bp = GameObject.Find("UI_Canvas").GetComponents<ButtonPromptDisplay>()[0];
         bp2 = GameObject.Find("UI_Canvas").GetComponents<ButtonPromptDisplay>()[1];
+        interactionIcon = GameObject.Find("UI_Canvas").GetComponents<InteractionIcon>()[0];
```
Comment (Lakshya2610) [inline Assets/Scripts/PlayerController.cs]: Create a string in Globals for "UI_Canvas" and replace this & other instances. Having raw strings in script is against the style convention we're following

File: Assets/Scripts/UIElements/InteractionIcon.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    public Texture handImg;
+    public Texture eyeImg;
+    public RawImage iconImg;
+    public GameObject interactionIcon;
+    
+    void Start()
+    {
+        interactionIcon.SetActive(false);
+    }
+    public void showReflectionIcon()
```
Comment (Lakshya2610) [inline Assets/Scripts/UIElements/InteractionIcon.cs]: Same style error with function names in this script as well. Function names should start with capital letter

**Review by Lakshya2610 — CHANGES_REQUESTED** [review]
Looks good, there are some styling issues that need to be fixed (check comments). After that it should be good to merge

**Pull request merged**